.github/workflows/scheduled.yaml

```diff
@@ -114,6 +123,17 @@ jobs:
           : Check Nightly Runs ☑️
           if [[ "${RUNNER_DEBUG}" ]]; then set -x; fi
 
+          # This 'gh' command retrieves the last 2 runs of the workflow defined
+          # by 'scheduled.yaml' and retrieve only the 'headSha' value of the
+          # JSON response payload.
+          #
+          # As this job runs in context of the same workflow, the first element
+          # of the workflow list will be the currently active run.
+          #
+          # The jq expression then selects the 'headSha' element of the second
+          # element in the array, thus the sha-1 hash of the commit used for
```

Comment:
This seems like an awkward use of "thus". I would just use, "which should be" or "which will be".